test/actions.test.ts

```diff
@@ -2,68 +2,109 @@ import * as chai from "chai";
 import * as chaiAsPromised from "chai-as-promised"
 import * as dotenv from "dotenv";
 import { ethers } from "ethers";
+import { createInstance } from "../src";
 
 import * as actions from "../src/actions";
-import { getSaleStatus, getWhitelist } from "../src/actions";
-import { getWhiteListSaleContract } from "../src/contracts";
-import { Claim, IPFSGatewayUri, SaleStatus } from "../src/types";
+import { getSaleStatus, getMintlist } from "../src/actions";
+import { getWolfSaleContract, WolfSale } from "../src/contracts";
+import { Claim, Config, Instance, Maybe, Mintlist, SaleStatus } from "../src/types";
 
 const expect = chai.expect;
 chai.use(chaiAsPromised.default);
 dotenv.config();
 
 describe("Test Custom SDK Logic", () => {
-  // Kovan
-  const whitelistSimpleSaleContractAddress =
-    "0xa6A3321b743C31912263090275E24d8b1A50cFE8";
   const provider = new ethers.providers.JsonRpcProvider(
-    process.env["INFURA_URL"]
+    process.env["INFURA_URL"],
+    4
```

Comment:
Why lock the network to `4`? If it's because you only want this test running on Rinkeby should we maybe change the env variable `INFURA_URL` to `INFURA_URL_RINKEBY`?

Reply:
good point